external-crates/move/crates/move-compiler/tests/move_2024/parser/clever_assertion_invalid_annotations.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: crates/move-compiler/tests/move_check_testsuite.rs
+info:
+  flavor: core
+  edition: 2024.alpha
+  lint: false
+---
+error[E10003]: invalid attribute value
+  ┌─ tests/move_2024/parser/clever_assertion_invalid_annotations.move:4:18
+  │
+4 │     #[error(code=1024)]
+  │                  ^^^^ Error code must be a u8
```

Comment:
We should copy the error here for inferred nums, specifying that this is too large